.github/workflows/rust-ci.yml

```diff
@@ -21,23 +21,17 @@ jobs:
           submodules: recursive
```

Comment:
I recommend adding a .NET workflow that runs your tests, similar to this .NET workflow https://github.com/eclipse-ibeji/ibeji-example-applications/blob/main/.github/workflows/dotnet-ci.yml.

Also I recommend adding CodeQL to your security audit, similar to this workflow https://github.com/eclipse-ibeji/ibeji-example-applications/blob/main/.github/workflows/security-audit.yml, for the .NET code too

Reply:
I have added the CodeQL wokflow in a separate merge.  Good suggestion!

We do not need unit testing for the small amount of C# code that has been added.  This can be added later when we have something more substantial than a CLI wrapper.